src/components/Form.vue

```diff
@@ -4,16 +4,14 @@
       @submit.prevent="submitAction"
     >
       <input-generator
-        v-for="element in elements" :key="element.id"
+        v-for="element in elements"
+        :key="element.id"
         :element="element"
         :data-field="element.id"
         v-model="element.value"
         :class="`form__element form__element--${element.type}`"
       />
-      <input-generator
-        :class="'form__element--submit'"
-        :element="submitButton"
-      />
+      <button @click="submitAction">action</button>
```

Comment:
Why man? @volcanioo

Reply:
it's a temporary solution to avoid console errors